### DISCUSSION TITLE
Is it necessary to dived by 200 first and times 200 behind？

### DISCUSSION THREAD

**Discussion by zyc573823770** [open-mmlab/mmpose#1339] [Q&A] [answered]
Hi, great work! However when I read detail about code processing data.  I wonder is it necessary to dived by 200 here
and what's the meaning of 'pixel std'
https://github.com/open-mmlab/mmpose/blob/11a3e35e21cb202c129351be936f5398e34d1c10/mmpose/datasets/datasets/base/kpt_2d_sview_rgb_img_top_down_dataset.py#L153

and times 200 here, it is a bit redundant
https://github.com/open-mmlab/mmpose/blob/11a3e35e21cb202c129351be936f5398e34d1c10/mmpose/core/post_processing/post_transforms.py#L226

**Comment by jin-s13** [comment, marked as answer]
Because some datasets (mpii) directly provide scale/center, and they do not provide bbox. To make the whole pipeline consistent, we first divide it by 200.
